.github/.kubescape/exceptions.json

```diff
@@ -14,5 +14,167 @@
         }
       }
     ]
+  },
+  {
+    "name": "application-credentials-in-configuration-files",
```

Comment:
pls disable this rule only where it comes up now so that we can catch new actual positives in the future for other resources